src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.Arrays;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
```

Comment:
Don't begin class or method implementation with an empty line. The empty line at the start of the class definition (line 6) should be removed.